src/sentry/utils/redis.py

```diff
@@ -287,6 +278,14 @@ def call_script(client, keys, args):
             script[0].registered_client = None
         return script[0](keys, args, client)
 
+    call_script.__doc__ = u"""Executes {!r} as a Lua script on a Redis server.
+
+Takes the client to execute the script on as the first argument,
+followed by the values that will be provided as ``KEYS`` and ``ARGV``
+to the script as two sequence arguments.""".format(
+        path
+    )
```

Comment:
kinda gross but yeah

Reply:
Maybe a good alternative would be:

```python
call_script.__doc__ = call_script.__doc__.format(path)
```

And keep the docstring in place so it's together with the function, just without formatting in place.